htdocs/core/modules/project/doc/doc_generic_project_odt.modules.php

```diff
@@ -527,8 +550,12 @@ function write_file($object,$outputlangs,$srctemplatepath)
 				$array_thirdparty=$this->get_substitutionarray_thirdparty($socobject,$outputlangs);
 				$array_objet=$this->get_substitutionarray_object($object,$outputlangs);
 				$array_other=$this->get_substitutionarray_other($outputlangs);
+                // retrieve contact information for use in project as contact_xxx tags
+        		$array_thirdparty_contact = array();
+        		if ($usecontact)
+            			$array_thirdparty_contact=$this->get_substitutionarray_contact($contactobject,$outputlangs,'contact');
```

Comment:
Contacts for project must be clearly managed differently than contacts of thirdparties. Thay may be differents.
Here itt seems here the contact you get is not the contact of the thirdparty but the contact of the project (PROJECTLEADER).
So, you should rename $array_thirdparty_contact into $array_project_contact